src/settings-ui/Settings.UI/SettingsXAML/Views/ShellPage.xaml

```diff
@@ -105,135 +105,158 @@
-
-                <NavigationViewItem
-                    x:Uid="Shell_ImageResizer"
-                    helpers:NavHelper.NavigateTo="views:ImageResizerPage"
-                    Icon="{ui:BitmapIcon Source=/Assets/Settings/Icons/ImageResizer.png}" />
-
-                <NavigationViewItem
-                    x:Uid="Shell_KeyboardManager"
-                    helpers:NavHelper.NavigateTo="views:KeyboardManagerPage"
-                    Icon="{ui:BitmapIcon Source=/Assets/Settings/Icons/KeyboardManager.png}" />
-
-                <NavigationViewItem
-                    x:Uid="Shell_MouseUtilities"
-                    helpers:NavHelper.NavigateTo="views:MouseUtilsPage"
-                    Icon="{ui:BitmapIcon Source=/Assets/Settings/Icons/MouseUtils.png}" />
-
-                <NavigationViewItem
-                    x:Uid="Shell_MouseWithoutBorders"
-                    helpers:NavHelper.NavigateTo="views:MouseWithoutBordersPage"
-                    Icon="{ui:BitmapIcon Source=/Assets/Settings/Icons/MouseWithoutBorders.png}" />
-
-                <NavigationViewItem
-                    x:Uid="NewPlus_Product_Name"
-                    helpers:NavHelper.NavigateTo="views:NewPlusPage"
-                    Icon="{ui:BitmapIcon Source=/Assets/Settings/Icons/NewPlus.png}" />
-
-                <NavigationViewItem
-                    x:Uid="Shell_Peek"
-                    helpers:NavHelper.NavigateTo="views:PeekPage"
-                    Icon="{ui:BitmapIcon Source=/Assets/Settings/Icons/Peek.png}" />
-
-                <NavigationViewItem
-                    x:Uid="Shell_PowerRename"
-                    helpers:NavHelper.NavigateTo="views:PowerRenamePage"
-                    Icon="{ui:BitmapIcon Source=/Assets/Settings/Icons/PowerRename.png}" />
-                <NavigationViewItem
-                    x:Uid="Shell_PowerLauncher"
-                    helpers:NavHelper.NavigateTo="views:PowerLauncherPage"
-                    Icon="{ui:BitmapIcon Source=/Assets/Settings/Icons/PowerToysRun.png}" />
-
-                <NavigationViewItem
-                    x:Uid="Shell_QuickAccent"
-                    helpers:NavHelper.NavigateTo="views:PowerAccentPage"
-                    Icon="{ui:BitmapIcon Source=/Assets/Settings/Icons/QuickAccent.png}" />
-
-                <NavigationViewItem
-                    x:Uid="Shell_RegistryPreview"
-                    helpers:NavHelper.NavigateTo="views:RegistryPreviewPage"
-                    Icon="{ui:BitmapIcon Source=/Assets/Settings/Icons/RegistryPreview.png}" />
-
-                <NavigationViewItem
-                    x:Uid="Shell_MeasureTool"
-                    helpers:NavHelper.NavigateTo="views:MeasureToolPage"
-                    Icon="{ui:BitmapIcon Source=/Assets/Settings/Icons/ScreenRuler.png}" />
+                    x:Uid="Shell_TopLevelSystemTools"
+                    Icon="Page2"
+                    SelectsOnInvoked="False">
+                    <NavigationViewItem.MenuItems>
```

Comment:
Except of Awake and ShortcutGuide this tools feel more like advanced tools instead of (os) system tools.